bundle/manifests/gatekeeper-operator.clusterserviceversion.yaml

```diff
@@ -543,7 +543,7 @@ spec:
   provider:
     name: Red Hat
   relatedImages:
-  - image: openpolicyagent/gatekeeper:v3.14.0
+  - image: openpolicyagent/gatekeeper:v3.15.1
     name: gatekeeper
-  replaces: gatekeeper-operator.v3.11.1
-  version: "3.14.0"
+  replaces: gatekeeper-operator.v3.14.1
```

Comment:
Should this be 3.14.2 now?

Reply:
We're working on releasing `v3.14.1`--`v3.14.2` doesn't exist, so I think this is correct?

Reply:
Oh right... it's v3.11 that has a v3.11.2.